### ISSUE TITLE
Hopper

### ISSUE THREAD

**Issue by doczoidberg** [Hitman666/nbCcTalkCoinAcceptor#4]
is it possible to integrate a hopper with this project? I want a hopper such as http://innovative-technology.com/products/products-main/210-smart-hopper

**Comment by sarpakal** [comment]
Yes. It can be integrated. It needs a lot of afford, since the project is
not complete.
Meanwhile, some special functions of Smart Hopper may not be used. It
should be pre-investigated if your project needs those ones.

Kind regards,

Sarp Akal
Payment Systems Expert


On Mon, Feb 15, 2016 at 12:08 PM, Daniel Zuidinga <notifications@github.com>
wrote:

> is it possible to integrate a hopper with this project? I want a hopper
> such as
> http://innovative-technology.com/products/products-main/210-smart-hopper
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Hitman666/nbCcTalkCoinAcceptor/issues/4>.
>



-- 
Sarp Akal
